docs/Makefile

```diff
@@ -19,7 +19,7 @@ docs-test: pull
 
 sources/installation/helm/reference.md: ../production/helm/loki/reference.md.gotmpl
 ifeq ($(BUILD_IN_CONTAINER),true)
-	docker run --rm --volume "$(realpath ..):/helm-docs:z" -u "$$(id -u)" "docker.io/jnorwood/helm-docs:v1.8.1" \
```

Comment:
generate doc was failing with this version. Updating the version solved the issue.

```
make -C docs sources/installation/helm/reference.md

time="2022-11-21T09:55:35Z" level=info msg="Found Chart directories [loki]"
time="2022-11-21T09:55:35Z" level=info msg="Generating README Documentation for chart /helm-docs/production/helm/loki"
time="2022-11-21T09:55:35Z" level=warning msg="Error generating documentation for chart /helm-docs/production/helm/loki: template: /helm-docs/production/helm/loki:38:19: executing \"chart.valuesTableHtml\" at <{{template \"chart.valueDefaultColumnRender\" .}}>: template \"chart.valueDefaultColumnRender\" not defined"
mv "../production/helm/loki/reference.md" "sources/installation/helm/reference.md"

```